sql/core/src/main/scala/org/apache/spark/sql/execution/CacheManager.scala

```diff
@@ -126,7 +126,9 @@ class CacheManager extends Logging with AdaptiveSparkPlanHelper {
     if (storageLevel == StorageLevel.NONE) {
       // Do nothing for StorageLevel.NONE since it will not actually cache any data.
     } else if (lookupCachedDataInternal(normalizedPlan).nonEmpty) {
-      logWarning("Asked to cache already cached data.")
+      logWarning(log"An attempt was made to cache data even though the data had already been " +
```

Comment:
Again, this is very similar to the changes in https://github.com/apache/spark/pull/45990/files#diff-88635a13b65f19dcc80b865d903b498b8328607f96c088402a8ebdbb857eedf9R303. I don't think we should have two duplicated logs with different log level.

Reply:
@gengliangwang The log entry is not added as part of this PR, so I don't follow your concern.